core/lib/types/src/eth_sender.rs

```diff
@@ -9,6 +28,7 @@ pub struct EthTx {
     pub tx_type: AggregatedActionType,
     pub created_at_timestamp: u64,
     pub predicted_gas_cost: u64,
+    pub blob_sidecar: Option<EthTxBlobSidecar>,
```

Comment:
can you also add the info about this to the fmt below (so that it mentions how many blobs we have)